hrs-portlets-webapp/src/main/java/edu/wisc/portlet/hrs/web/benefits/BenefitInformationController.java

```diff
@@ -67,9 +68,9 @@ public String viewBenefitInfo(ModelMap model, PortletRequest request) {
       model.addAttribute("tab", tab);
       boolean isMadisonUser = !StringUtils.isBlank(userInfo.get("wiscEduSORName"));
       model.addAttribute("isMadisonUser", isMadisonUser);
-      
+      final PortletPreferences preferences = request.getPreferences();
+      model.addAttribute("learnMoreEBenefitGuide", preferences.getValue("ebenefitguidetext", null));
```

Comment:
IIRC it's coerced to the string `"null"` when displayed in `JSP`. It might be that the modern `EL` behaves differently than `JSTL` `<c:out`, tho. I could be wrong.